### PR TITLE
magento/devdocs#: Docs: Add documentation for related/upsell/crosssell products (DEVELOP)

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) provides docs for the https://github.com/magento/graphql-ce/issues/766 issue.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/products.html
- https://devdocs.magento.com/guides/v2.3/graphql/queries.html

## Links to Magento source code

- ...

Thank you!

CC: @keharper @rogyar 

whatsnew
Related items, up-sell, and cross-sell products are now supported in the [products query](https://devdocs.magento.com/guides/v2.3/graphql/reference/products.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/reference/products.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ Attribute | Data type | Description
 `categories` | [CategoryInterface] | The categories assigned to the product. See [categories endpoint]({{ page.baseurl }}/graphql/reference/categories.html) for more information
 `country_of_manufacture` | String | The product's country of origin
 `created_at` | String | Timestamp indicating when the product was created
+`crosssell_products` | [[ProductInterface](#ProductInterface)] | Cross-sell product(s)
```
Comment (rogyar) [inline guides/v2.3/graphql/reference/products.md]: I would suggest clarifying the description slightly
```suggestion
`crosssell_products` | [[ProductInterface](#ProductInterface)] | An array of cross-sell products
```
This suggestion is fair for all types of related products

Comment (atwixfirster) [reply guides/v2.3/graphql/reference/products.md]: **implemented**

Thanks, @rogyar

File: guides/v2.3/graphql/queries.md [hunk 2]
```diff
@@ -475,4 +475,105 @@ This query searches for products that have `name` that ends with `Short` or has
 }
 ```
 
-The query returns 14 item.
+The query returns 14 items.
+
+#### Retrieve related products, Up-sells and Cross-sells
```
Comment (keharper) [inline guides/v2.3/graphql/queries.md]:
```suggestion
#### Retrieve related products, up-sells and cross-sells 
```

File: guides/v2.3/graphql/queries.md [hunk 3]
```diff
@@ -475,4 +475,105 @@ This query searches for products that have `name` that ends with `Short` or has
 }
 ```
 
-The query returns 14 item.
+The query returns 14 items.
+
+#### Retrieve related products, Up-sells and Cross-sells 
+
+The following query shows how to get related products, Up-sells and Cross-sells for the particular product:
```
Comment (keharper) [inline guides/v2.3/graphql/queries.md]:
```suggestion
The following query shows how to get related products, up-sells, and cross-sells for a specific product:
```

Comment (atwixfirster) [reply guides/v2.3/graphql/queries.md]: **fixed**

Thank you, @keharper !

File: guides/v2.3/graphql/queries.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/queries.md]: **fixed**

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks as always, @atwixfirster

**Comment by atwixfirster** [comment]
> Thanks as always, @atwixfirster

welcome as always , @keharper !

Thank you very much!

**Pull request merged**